### DISCUSSION TITLE
Skin temperature and radiative temperature confusion in the single layer urban canopy model (SLUCM)

### DISCUSSION THREAD

**Discussion by cenlinhe** [NCAR/hrldas#79] [Q&A]
https://github.com/NCAR/hrldas/blob/8732faf7b68411a04cdf664dcf3e5e35d465121c/urban/wrf/module_sf_urban.F#L1596

This TS is the radiative temperature which should be differentiated from the skin temperature: 

https://github.com/NCAR/hrldas/blob/master/urban/wrf/module_sf_urban.F#L1541

This needs to be clarified in the future code

**Comment by tslin2** [comment]
Does this need to change to TSK, so it's more consistent with Noah-MP?
Also, does it make sense to calculate bulk urban temperature from road, wall, and roof as urban TSK instead of this equation?

**Reply by cenlinhe** [reply]
Let's think about this more before making any changes. I will come back to this issue in a few weeks when I have more time.

**Comment by tslin2** [comment]
https://github.com/DanLi-BU/WRF/issues/5#issue-1974926011

**Comment by tslin2** [comment]
related papers and methods:

https://iopscience.iop.org/article/10.1088/1748-9326/9/5/055001
![image](https://github.com/user-attachments/assets/8e842e50-bfc2-4781-aca7-167a4e11cb53)


https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2013JD021101
![image](https://github.com/user-attachments/assets/6a65973c-c061-4f44-b4e0-ac6b28410c49)


https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2013JD021227
![image](https://github.com/user-attachments/assets/8ec89ec3-0285-440d-a757-d3e36260513d)
